### DISCUSSION TITLE
How do i use the bot to post faq on covid 19 test kits?

### DISCUSSION THREAD

**Discussion by ghost** [ASEAN-Build-The-Earth/Merlion#48] [Q&A] [answered]
Im trying to post FAQs for Covid 19 test kits using the commands folder. I dont quite get it. I dont know how. Can someone help me?

**Comment by StoneMcYT** [comment, marked as answer]
Have you tried googling or looking at the API doc?
https://covid19api.com/

**Comment by ghost** [comment]
sorry, i looked at my code and i found out that everything is correct basically what i did is do the normal formatting of the command and then i put down an alias to make sure that when u send it they know what you are talking about, and then send the msg. Sorry for wasting your time
